utils/get-all-articles.js

```diff
@@ -0,0 +1,11 @@
+import axios from "axios";
+
+function getAllArticles() {
+  return axios
+    .get(`https://nc-news-backend-project-u3fr.onrender.com/api/articles`)
+    .then((response) => {
+      return response.data.articles;
```

Comment:
Remember when you get further on and have multiple different requests that you can create an _axios instance_ which will make a reusable host URL